data/models/webinar.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+
+
+class Webinar(models.Model):
+    class Meta:
+        verbose_name = "webinaire"
+        ordering = ["start_date"]
+
+    creation_date = models.DateTimeField(auto_now_add=True)
+    modification_date = models.DateTimeField(auto_now=True)
+
+    title = models.TextField(verbose_name="titre")
+    start_date = models.DateTimeField(verbose_name="date de début")
+    end_date = models.DateTimeField(verbose_name="date de fin")
+    tagline = models.TextField(blank=True, default="", verbose_name="description courte")
+    link = models.TextField(verbose_name="lien pour s'inscrire")
+
```

Comment:
La méthode `clean` sert à effectuer une validation après l'utilisation d'un form, comme dans l'admin. Cette partie de la [doc Django](https://docs.djangoproject.com/en/5.0/ref/forms/validation/#cleaning-and-validating-fields-that-depend-on-each-other) l'explique bien même s'ils parlent des `Forms`, mais c'est applicable aussi au niveau du modèle

> We are performing validation on more than one field at a time, so the form’s clean() method is a good spot to do this.